actions/create-pr/README.md

```diff
@@ -36,6 +36,13 @@ jobs:
   create_pr:
     runs-on: ubuntu-latest
     steps:
+      - name: 'Create GitHub App Token'
```

Comment:
I also updated the example in the documentation for how to use the create PR action to include the GitHub app token generation